src/models.py

```diff
@@ -74,6 +74,7 @@ class Book(db.Model):
     updated_at = db.Column(db.DateTime, onupdate=datetime.now())
     is_available = db.Column(db.Boolean, default=True)
     loan_points = db.Column(db.Integer)
+    url = db.Column(db.String)
```

Comment:
what url? image url? image key? does cloudinary return a url or an identifier for the image?

Reply:
This should be renamed as `image_url`